src/Qir/Execution/CommandLineCompiler/CommandLineCompiler.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
The last thing to do to make the project self-contained, would be to add the `<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>` property. This copies the dependencies to the output folder, which is what we want for the `Microsoft.Quantum.Simulators` and `Microsoft.Quantum.Qir.Runtime` packages.

Reply:
I would also probably solve this as part of packaging rather than as using CopyLocalLockFileAssemblies here; I believe it will work better with our build (but am happy to be convinced otherwise).

Reply:
Yes, this was before deciding to use `PackAsTool`. `CopyLocalLockFileAssemblies` is not meant for packaging but to make the project self-contained in case something like `dotnet publish` was used. So, I think it's ok to remove `CopyLocalLockFileAssemblies` from the project file.

Reply:
removed